src/input/input.directive.ts

```diff
@@ -17,3 +17,13 @@ import { Directive, HostBinding } from "@angular/core";
 export class TextInput {
 	@HostBinding("class") inputClass = "bx--text-input";
 }
+
+
+@Directive({
+	selector: "[ibmTextArea]"
+})
+export class TextAreaInput {
+	@HostBinding("class.bx--text-area") baseClass = true;
+	@HostBinding("class.bx--text-area--v2") secondaryClass = true;
```

Comment:
do we need both `text-area` and `text-area--v2`? I can't seem to find a reference to `text-area--v2` in the carbon source.

Reply:
Not sure- just seemed like both were applied in the carbon storybook so I applied both. I will double check.